xonsh/lexer.py

```diff
@@ -59,7 +59,7 @@ def token_map():
     tm[NEWLINE] = 'NEWLINE'
     tm[INDENT] = 'INDENT'
     tm[DEDENT] = 'DEDENT'
-    if PYTHON_VERSION_INFO >= (3, 5, 0):
+    if (3, 5, 0) <= sys.version_info < (3, 7, 0, 'beta', 4):
```

Comment:
I'd prefer if this used the `PYTHON_VERSION_INFO` variable and only compared against ` < (3, 7, 0)`, since in the long run it is unlikely that anyone will be installing versions of 3.6.0 < Python <= 3.7.0